content/docs/tools/cargo-flash.md

```diff
@@ -36,7 +36,7 @@ cargo flash --release --chip <chip_name>
 
 # Don't know if your target is supported
 # by cargo flash and what it's name is?
-cargo flash --list-chips
+probe-rs info
```

Comment:
I think `probe-rs chip list` would be a better match, `probe-rs info` will just show some information about the debug interface of a connected chip, but you can't figure out the target name with it.

Reply:
sounds good to me, changed, pushed!